pyartcd/pyartcd/pipelines/scan_fips.py

```diff
@@ -0,0 +1,64 @@
+"""
+For this command to work, https://github.com/openshift/check-payload binary has to exist in PATH and run as root
+This job is deployed on ART cluster
+"""
+import json
+import click
+from typing import Optional
+from pyartcd.runtime import Runtime
+from pyartcd.cli import cli, pass_runtime, click_coroutine
+from pyartcd import exectools
+
+
+class ScanFips:
+    def __init__(self, runtime: Runtime, version: str, nvrs: Optional[list]):
+        self.runtime = runtime
+        self.version = version
+        self.nvrs = nvrs
+
+        # Setup slack client
+        self.slack_client = self.runtime.new_slack_client()
+        self.slack_client.bind_channel(f"openshift-{self.version}")
+
+    async def run(self):
```

Comment:
can we import doozer lib to use the function directly?

Reply:
Hmm @Ximinhan I'm not sure if I follow you? Which part of the code can we get from doozerlib? :thinking:

Reply:
I think Ximin might be suggesting to do something like:

```
    async def run(self):
        fips_pipeline = doozerlib.cli.scan_fips.ScanFipsCli(runtime=runtime,
                                                                                                 nvrs=nvrs.split(",") if self.nvrs else None)
        fips_pipeline.run()
```

Instead of running Doozer in a separate process, we could import doozerlib and run it directly. Only thing is, `doozerlib.cli.scan_fips.ScanFipsCli.run()` does not return anything, but prints to stdout. So we'd need to change that. Although I am not sure this is worth the pain, we generally call Doozer in its own process from pyartcd.

Reply:
Hmm I haven't tried that out before, but if you guys think its worth it, I can work on the change

Reply:
yes, I mean maybe we can import ScanFipsCli here, ScanFipsCli(runtime=runtime,nvr=self.nvrs)
the pr is going to run "check-payload scan image --spec" this cli it doesn't seems to be need tightly bonded with doozer, can we directly call it here?

Reply:
> this cli it doesn't seems to be need tightly bonded with doozer, can we directly call it here
You're right. But since the current pattern in our code base is to create a doozer command then calling the art-cd command, didn't want to break that pattern.

Reply:
> directly use pull spec can save some effort to get the build object again in the job

Is this scenario when humans run the command? Cause in that case the fastest would be to directly use the `check-payload scan image --spec <pull-spec>` command right.